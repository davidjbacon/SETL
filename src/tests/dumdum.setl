a := +/[c('[a-z]') ? '' : c IN +/[char(i): i IN [1..255]]];
b := +/[c+c: c IN a];
b := 3*a;
print(b);
LOOP FOR i IN [1..52] DO
  j := (i-1) MOD #a + 1;
  k := j MOD #a + 1;
  b(1) := '';
print(b);
END LOOP;
PRINT(b);
