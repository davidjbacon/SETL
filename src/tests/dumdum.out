abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
bcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
cdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
defghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
efghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
fghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
ghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
hijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
ijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
jklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
klmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
lmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
mnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
nopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
opqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
pqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
qrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
rstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
stuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
tuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
uvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
vwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
wxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
xyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
yzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
zabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
bcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
cdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
defghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
efghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
fghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
ghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
hijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
ijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
jklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
klmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
lmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
mnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
nopqrstuvwxyzabcdefghijklmnopqrstuvwxyz
opqrstuvwxyzabcdefghijklmnopqrstuvwxyz
pqrstuvwxyzabcdefghijklmnopqrstuvwxyz
qrstuvwxyzabcdefghijklmnopqrstuvwxyz
rstuvwxyzabcdefghijklmnopqrstuvwxyz
stuvwxyzabcdefghijklmnopqrstuvwxyz
tuvwxyzabcdefghijklmnopqrstuvwxyz
uvwxyzabcdefghijklmnopqrstuvwxyz
vwxyzabcdefghijklmnopqrstuvwxyz
wxyzabcdefghijklmnopqrstuvwxyz
xyzabcdefghijklmnopqrstuvwxyz
yzabcdefghijklmnopqrstuvwxyz
zabcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
abcdefghijklmnopqrstuvwxyz
